src/runtime/createRunTime.tsx

```diff
@@ -651,11 +723,39 @@ function CreateRunTime({
   };
   const handleCancelButton = () => {
     setOpenCreateTemplate(false);
+    const content = new AuthLogin(themeManager);
```

Comment:
I thought this widget is a child of configselection which is a child of authlogin?   I don't think this is correct, either unwind the stack or call the jupyter command if you want to open up a new widget.

Reply:
resolved

Reply:
we have removed this line from the above code